lmfdb/utils/utilities.py

```diff
@@ -87,7 +87,10 @@ def list_factored_to_factored_poly_otherorder(sfacts_fc_list, galois=False, vari
                     gtoprint[(val, i)] = elt/p**val
         glatex = latex(ZZpT(gtoprint))
         if  e > 1:
-            outstr += '( %s )^{%d}' % (glatex, e)
+            if len(glatex) != 1:
+                outstr += '( %s )^{%d}' % (glatex, e)
+            else:
+                outstr += '%s^{%d}' % (glatex, e)
```

Comment:
I'm not 100% sure that changing this method won't affect anything else in the codebase. It looks like it gets used in `galois_pretty_print` in the lfunctions folder, which only ends up getting used in `/TensorProducts/`, so I'm reasonably confident, but could do with a second opinion (especially since I know nothing about the `TensorProducts` area)

Reply:
I don't think TensorProducts is currently used.

Reply:
it looks good to me.

Reply:
Great, thanks!